core/src/ast/query.rs

```diff
@@ -1,5 +1,5 @@
 use {
-    super::{Expr, IndexOperator},
+    super::{Expr, IndexOperator, ToSqlUnquoted},
```

Comment:
In `ast/query.rs`, `ToSqlUnquoted` impl for `IndexItem` is missing.
It looks that only except `IndexItem`, all looks well implemented.

Could you also add some unit tests for new added lines?